frontend/src/App.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import WelcomePanel from './components/panels/home_panel/WelcomePanel';
+import ProjectListPanel from './components/panels/project_list_panel/ProjectListPanel';
+import ProjectFiltersPanel from './components/panels/project_filters_panel/ProjectFiltersPanel';
```

Comment:
Per my comment on structuring your files this would be
```javascript
import ProjectListPanel from './components/panels/ProjectListPanel'`
```

I believe you could also put an `index.js` inside `components/panels` as follows

```javascript
export {
    ProjectListPanel,
    ProjectFilterPanel
}
```

And then do

```javascript
import { ProjectListPanel, ProjectFilterPanel } from './components/panels';
```

Reply:
Also I wouldn't put the sub components for `ProjectListPanel` in their own directory I'd just put them under `ProjectListPanel/`. So it would be something like

`ProjectListPanel/index.js` <- Defines ProjectListPanel
`ProjectListPanel/OtherComponent.js` <- Defines component used in `index.js`